api/Document.json

```diff
@@ -5874,13 +5874,60 @@
           }
         }
       },
+      "parseHTML_static": {
+        "__compat": {
+          "description": "`parseHTML` static method",
+          "spec_url": "https://wicg.github.io/sanitizer-api/#dom-document-parsehtml",
+          "tags": [
+            "web-features:parse-html-unsafe",
+            "web-features:html-sanitizer-api"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "138",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.security.sanitizer.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror",
+            "webview_ios": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
       "parseHTMLUnsafe_static": {
         "__compat": {
           "description": "`parseHTMLUnsafe()` static method",
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/parseHTMLUnsafe_static",
           "spec_url": "https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-parsehtmlunsafe",
           "tags": [
-            "web-features:parse-html-unsafe"
+            "web-features:parse-html-unsafe",
```

Comment:
Note, this is in the new spec where it takes a sanitizer. However the version with the sanitizer it is not yet implemented anywhere, so I have not added the spec. 
Should I add the spec, a partial implementation, and a note that the sanitizer option is not yet supported?

My feeling is "not until implemented", but wanted to be sure of the policy.

This is also true for `Element.setHTMLUnsafe`

Reply:
Closing. Not implemented anywhere yet. Will add spec when the new version is added.